app/views/templates/payments/PaymentsHistoryTabsContent.scala.html

```diff
@@ -17,83 +17,57 @@
 @import config.AppConfig
 @import models.viewModels.PaymentsHistoryModel
 
-@this(paymentsHistoryCharge: PaymentsHistoryCharge)
+@this(govukTable: GovukTable,
+      paymentsHistoryCharge: PaymentsHistoryCharge)
 
 @(years: Seq[Int],
   transactions: Seq[PaymentsHistoryModel],
-  showPreviousPaymentsTab: Boolean,
-  javascriptEnabled: Boolean)(implicit messages: Messages, appConfig: AppConfig, user: User)
-
-@if(!javascriptEnabled) {
-  <p>
-    @years.map { year =>
-      <a href="#nonJS-@year">@year</a> @if(years.lastOption.getOrElse(0) != year){|}
-    }
-
-    @if(showPreviousPaymentsTab) {
-      | <a href="#nonJS-previous-payments">@messages("paymentsHistory.previousPayments.heading")</a>
-    }
-  </p>
-}
+  showPreviousPaymentsTab: Boolean
+  )(implicit messages: Messages, appConfig: AppConfig, user: User)
```

Comment:
Why has the `javascriptEnabled` boolean and the non-JS elements of the page been removed? We should still support a non-JS version of this page.